layers/+tools/google-calendar/packages.el

```diff
@@ -0,0 +1,89 @@
+;;; packages.el --- google-calendar layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Markus Johansson <markus.johansson@me.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+(defconst google-calendar-packages
+  '(org-gcal
+    calfw))
+
+(defun google-calendar/init-org-gcal ()
+  "Initializes org-gcal and adds keybindings for it's exposed functions"
+  (use-package org-gcal
+    :init
+    (progn
```

Comment:
There's a [bug in org-gcal](https://github.com/myuhe/org-gcal.el/pull/81) that can cause it to choke on events that are private (Say you share your work calendar with your personal account and your work calendar has something marked private). 

This will work around it until it is fixed in org-gcal and may be worth including in this layer:

```
(push (lambda (event) (plist-get event :summary)) org-gcal-fetch-event-filters)
```